packages/govuk-frontend/src/govuk/components/checkboxes/_index.scss

```diff
@@ -252,16 +233,14 @@
     // (If you do use them, they won't look completely broken... but seriously,
     // don't use them)
     .govuk-checkboxes__hint {
-      padding: 0;
-      clear: both;
+      padding-left: $govuk-touch-target-size;
```

Comment:
Mind checking hint alignment for small sizes?

![Hint alignment issue](https://github.com/alphagov/govuk-frontend/assets/415517/3d70b4ff-672f-4047-9c75-f01758c148b8)